aws/resource_aws_ecs_cluster.go

```diff
@@ -30,11 +30,42 @@ func resourceAwsEcsCluster() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-			"tags": tagsSchema(),
 			"arn": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"capacity_providers": {
```

Comment:
This new attribute is missing documentation in `website/docs/r/ecs_cluster.html.markdown`